ArcGISRuntimeSDKQt_QMLSamples/Maps/ShowLocationHistory/ShowLocationHistory.qml

```diff
@@ -0,0 +1,115 @@
0.590808127,4040391.6174997534],[-13007272.447372077,4040321.745969433],[-13007314.138671933,4040238.363369706],[-13007231.528133322,4040312.0952055696],[-13007207.594238957,4040409.7609357936],[-13007180.186069606,4040449.908113437],[-13007177.483855726,4040397.4079580577],[-13007127.685914226,4040489.6692605256],[-13007139.652861413,4040315.1834499966],[-13007031.564306224,4040541.39735479],[-13007057.042322807,4040339.117344355],[-13007011.490717405,4040449.52208287],[-13006952.428042606,4040544.4855992175],[-13006995.277434127,4040379.2645219946],[-13006946.637584293,4040226.7824530574],[-13006908.806589976,4040240.2935224567],[-13006901.472009445,4040303.2165028006],[-13006867.501320671,4040238.363369685],[-13006879.854298407,4040164.245503266],[-13006821.563684715,4040195.1279476066],[-13006794.541545916,4040224.8523002844],[-13006779.4863543,4040166.5616865917],[-13006688.769174052,4040199.374283703],[-13006678.732379641,4040249.5582557567],[-13006600.75420769,4040344.5217721006],[-13006478.768552545,4040397.4079580316],[-13006460.625116494,4040465.3493355783],[-13006471.433972025,4040525.956132594],[-13006449.044199886,4040656.0484293764],[-13006472.592063695,4040680.7543848488],[-13006468.345727604,4040742.133242973],[-13006504.246569151,4040809.302559414],[-13006465.643513724,4040807.372406643],[-13006454.44862765,4040841.7291259686],[-13006497.298019174,4040861.416684236],[-13006470.66191093,4040884.9645480453],[-13006528.952524621,4040927.0418784595],[-13006488.805346979,4040929.7440923396],[-13006467.187635941,4040970.6633310863],[-13006510.809088573,4040986.1045532566],[-13006483.400919221,4041011.1965392837],[-13006509.26496636,4041093.4210473364],[-13006495.367866406,4041110.0203611697],[-13006503.474508055,4041175.6455553896],[-13006600.754207738,4041320.4070132314],[-13006637.813140953,4041406.8778573833],[-13006633.566804856,4041479.8376321383],[-13006606.158635503,4041526.9333597533],[-13006582.224741139,4041544.6907652495],[-13006537.05916629,4041570.554812385],[-13006571.801916175,4041578.6614540224],[-13006598.051993866,4041633.4777927273],[-13006550.57023569,4041631.161609402],[-13006480.69870537,4041677.871306458],[-13006382.26091403,4041783.2576477686],[-13006354.466714123,4041873.588797465],[-13006273.014267175,4041992.872238723],[-13006138.675634298,4042124.1226271675],[-13006090.421815023,4042234.527365678],[-13006031.359140223,4042273.9024822125],[-13005898.178598998,4042420.98012338],[-13005806.689357642,4042428.314703906],[-13005670.420571996,4042393.957984575],[-13005583.177666733,4042405.924931757],[-13005500.567128133,4042402.064626215],[-13005475.475142105,4042385.8513429365],[-13005446.522850536,4042364.2336318977],[-13005417.570558969,4042337.2114930963],[-13005398.269031256,4042312.891568178],[-13005336.504142575,4042238.3876712057],[-13005192.514745843,4042221.0162962615],[-13005034.628249153,4042147.6704909513],[-13004884.076332992,4042148.8285826137]]],"spatialReference":{"latestWkid":3857,"wkid":102100}}
```

Comment:
wrap this line

Reply:
this is the workflow you could use in C++ as well. A bit cleaner than creating another builder IMO